clientlibs/java/src/main/java/org/upgradeplatform/client/ExperimentClient.java

```diff
@@ -92,20 +90,20 @@ public void close() {
 
 	// Initialize user with userId
 	public void init(final ResponseCallback<InitializeUser> callbacks) {
-		InitializeUser initUser = new InitializeUser(this.userId, null, null);
+		InitializeUser initUser = new InitializeUser(null, null);
 		initializeUser(initUser, callbacks);
```

Comment:
In the `initializeUser` method, the request object is created with the response type `InitializeUser`, which should have `id` on it (but no longer does). Is it possible we need a new request bean for init?